mia/templates/mia-default/archive/system/etc/init.d/Y02firstboot

```diff
@@ -1,49 +1,50 @@
 #!/system/bin/sh
-# execute post-install script on First Boot
-# 01022012
+# Call `mia-firstboot.sh` if it exists.
+# @see https://github.com/CyanogenMod/android_vendor_cm/blob/cm-12.1/prebuilt/common/etc/init.d/90userinit
 
-FB_SCRIPT="/data/.firstboot.sh"
-TIMEOUT=5
+FB_SCRIPT="/data/local/mia-firstboot.sh"
```

Comment:
Actually, this is is the same location used for the CM `userinit.sh` and `userinit.d` scripts. And these scripts are very similar to the firstboot script, with the exception that the firstboot script is first waiting for the `com.android.systemui` process to finish.

https://github.com/CyanogenMod/android_vendor_cm/blob/cm-11.0/prebuilt/common/etc/init.d/90userinit
https://github.com/CyanogenMod/android_vendor_cm/blob/cm-12.1/prebuilt/common/etc/init.d/90userinit